content/en/synthetics/private_locations/_index.md

```diff
@@ -465,7 +485,74 @@ Because Datadog already integrates with Kubernetes and AWS, it is ready-made to
+1. Click **Next** on the welcome page, read the EULA, and accept the terms and conditions. Click **Next**.
+1. Modify where the application will be installed, or leave the default settings. Click **Next**.
+1. To configure your Windows private location, you can either:
+
+   - Paste and enter a JSON configuration for your Datadog Synthetics Private Location Worker. This file is generated by Datadog when you [create a private location][102].
+
+   {{< img src="synthetics/private_locations/configuration_selector_paste.png" alt="Synthetics Private Location Worker wizard, MSI installer. The option 'Paste in a JSON configuration' is selected. A text field for this JSON configuration is displayed." style="width:80%;" >}}
+
+   - Browse or type a file path to a file containing a JSON configuration for your Datadog Synthetics Private Location Worker. You can type the file path or leave it blank and run `synthetics-pl-worker.exe --config=<PathToYourConfiguration>` in the Windows command-line prompt after the installation is complete.
+  
+   {{< img src="synthetics/private_locations/configuration_selector_browse.png" alt="Synthetics Private Location Worker wizard, MSI installer. The option 'Browse for a JSON configuration on my computer' is selected. A file selector is displayed." style="width:80%;" >}}
+
+1. You can apply the following configuration options:
+   
+   {{< img src="synthetics/private_locations/settings.png" alt="Synthetics Private Location Worker wizard, MSI installer. Firewall and log settings are displayed." style="width:80%;" >}}
+
+   Apply firewall rules needed by this program to Windows Firewall
+   : Allow the installer to apply firewall rules on install and remove them on uninstall.
 
+   Apply rules to block reserved IPs in Windows Firewall
+   : Configure blocking rules for Chrome, Firefox, and Edge (if they are installed) and add rules to block reserved IP address ranges outbound in Windows Firewall.
+
+   Enable File Logging
+   : Allow the Synthetics Private Location Worker to log files in the installation directory.
+
+   Log Rotation Days
+   : Specifies how many days to keep logs before deleting them from the local system.
+
+   Logging Verbosity
+   : Specifies the verbosity of the console and file logging for the Synthetics Private Location Worker. Options for verbosity when performing an unattended `msiexec` installation include `-v` for error, `-vv` for information, and `-vvv` for debug.
+
+1. Click **Next** and **Install** to start the installation process. 
+   
+Once the process is complete, click **Finish** on the installation completion page.
+
+<div class="alert alert-warning">If you entered your JSON configuration, the Windows Service starts running using that configuration. If you did not enter your configuration, run <code>synthetics-pl-worker.exe --config=<PathToYourConfiguration></code> from a command prompt or use the <code>start menu</code> shortcut to start the Synthetics Private Location Worker.</div>
```

Comment:
> start menu

@jakepruitt what is this?

Reply:
For reference: https://datadoghq.atlassian.net/wiki/spaces/SYN/pages/2187296818/Installing+Synthetics+Private+Location+Worker+with+MSI+installer#Final-Steps